homeassistant/components/unifi/__init__.py

```diff
@@ -75,11 +75,45 @@ async def async_remove_config_entry_device(
 ) -> bool:
     """Remove config entry from a device."""
     hub = config_entry.runtime_data
-    return not any(
+
+    # Get MAC address from device connections
+    device_mac = None
```

Comment:
```suggestion
    mac = None
```
Using device as a prefix signals this is a unifi network device, where as at this point in time we do not know